Lib/fontTools/feaLib/ast.py

```diff
@@ -1119,11 +1119,14 @@ def asFea(self, indent=""):
 class MultipleSubstStatement(Statement):
     """A multiple substitution statement.
 
-    ``prefix``, ``glyph``, ``suffix`` and ``replacement`` should be lists of
-    `glyph-containing objects`_.
-
-    If ``forceChain`` is True, this is expressed as a chaining rule
-    (e.g. ``sub f' i' by f_i``) even when no context is given."""
+    Args:
+        prefix: a list of `glyph-containing objects`_.
+        glyph: a single glyph-containing object.
```

Comment:
In the line above you have wrapped `glyph-containing object` as inline code inside backticks, should you do it consistently for the following lines as well?

Reply:
This was deliberate. The first line is a link to explain what a glyph-containing object is; after that, the reader should know and doesn't need the explanation again.

Reply:
ok thanks!